meson.build

```diff
@@ -198,6 +198,10 @@ install_data('data/labwc.desktop', install_dir: get_option('datadir') / 'wayland
 
 install_data('data/labwc-portals.conf', install_dir: get_option('datadir') / 'xdg-desktop-portal')
 
+# TODO: move this to clients/meson.build after the labnag PR
+clients = files('clients/lab-sensible-terminal')
+install_data(clients, install_dir: 'bin')
```

Comment:
I have a feeling that hardcoding `bin` is wrong, there is likely something like `get_option(*bindir')` instead but I didn't check.

Reply:
Would be preferable if that is adjustable, yes :) Especially since some distros have completed the `/usr` merge.
And `get_option('bindir')` exists, but iirc needs to be combined with `get_option('prefix')`

Reply:
It will work yes, but depending on the tooling to create packages it will complain about the install to a symlink location.
I just had recently a case, where `bindir` didn't work as expected and [this](https://github.com/dunst-project/dunst/issues/1462#issuecomment-2851363909) was the proposed solution by meson upstream. But not sure if that would also apply here. I'm not that knowledgable with the possebilities of `meson`